pkg/config/jobs.go

```diff
@@ -266,6 +266,19 @@ type Postsubmit struct {
 	JenkinsSpec *JenkinsSpec `json:"jenkins_spec,omitempty"`
 }
 
+// RetriggerFailed defines the configuration for retrying failed prowjobs.
+type RetriggerFailed struct {
+	// UntilSuccess stops retries once a successful run is achieved.
+	// The Attempts field still limits the total number of retries.
+	RunAll bool `json:"run_all,omitempty"`
```

Comment:
I can't think of a case where the user wants to initiate all retries and keep the last try's results. How about keeping this behavior as the default? Will it also make sense to make it configurable?

Reply:
Requestor wanted to have 2 strategies. Default one is until_success so if you achieve success next retry won't happen. Of course it's respecting the attempts field so it won't go over them. The second requested strategy was run_all. If this bool is true, no matter what will be the result, it will run defined number of times. This is to have more sophisticated statistical analisys of the case.